test/integration/etcd/data.go

```diff
@@ -281,13 +281,29 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		gvr("flowcontrol.apiserver.k8s.io", "v1alpha1", "flowschemas"): {
 			Stub:             `{"metadata": {"name": "va1"}, "spec": {"priorityLevelConfiguration": {"name": "name1"}}}`,
 			ExpectedEtcdPath: "/registry/flowschemas/va1",
+			ExpectedGVK:      gvkP("flowcontrol.apiserver.k8s.io", "v1beta1", "FlowSchema"),
 		},
 		// --
 
 		// k8s.io/kubernetes/pkg/apis/flowcontrol/v1alpha1
 		gvr("flowcontrol.apiserver.k8s.io", "v1alpha1", "prioritylevelconfigurations"): {
 			Stub:             `{"metadata": {"name": "conf1"}, "spec": {"type": "Limited", "limited": {"assuredConcurrencyShares":3, "limitResponse": {"type": "Reject"}}}}`,
 			ExpectedEtcdPath: "/registry/prioritylevelconfigurations/conf1",
+			ExpectedGVK:      gvkP("flowcontrol.apiserver.k8s.io", "v1beta1", "PriorityLevelConfiguration"),
```

Comment:
It's important to note that a 1.19 server with P&F enabled will not be able to read this persisted v1beta1 data. That means a 1.19 server with P&F enabled should not be run in a multi-server cluster with 1.20+ servers.

Reply:
Good point.  While normal, it does not have to be such a sharp transition.  Should we make the storage version be v1alpha1?  There is currently no difference between v1alpha1 and v1beta1.

Reply:
we should not persist alpha data into etcd for consumers that have not opted into the alpha level of the feature

Reply:
Good point too.  So the next question is, should we make the storage version depend on the FeatureGate?

Reply:
Added a line to the release notes, not sure if multi-line release notes are accepted.